src/Autofac/Builder/ContainerBuildOptions.cs

```diff
@@ -39,12 +39,6 @@ public enum ContainerBuildOptions
         /// </summary>
         None = 0,
 
-        /// <summary>
-        /// Prevents inclusion of standard modules like support for
-        /// relationship types including <see cref="IEnumerable{T}"/> etc.
-        /// </summary>
-        ExcludeDefaultModules = 2,
```

Comment:
This is a breaking change. We could add an overload to `Build` takes the `DefaultServiceFlags` and passes through `None`.

Reply:
Agreed, an overload would be better.